pysat/tests/test_instrument.py

```diff
@@ -1179,7 +1181,7 @@ def test_list_comprehension(self):
         assert dates.is_monotonic_increasing
 
         # dates are unique
-        assert np.all(np.unique(dates) == dates.values)
+        assert len(np.unique(dates)) == len(insts)
```

Comment:
This fits the above comment better.  Each date in the list should be unique.  Was breaking due to different timestamp types